libs/sdk-ui-tests-e2e/README.md

```diff
@@ -6,37 +6,58 @@ This project contains End-to-End tests for GoodData.UI components.
 
 Tests in this library verify various go-through scenarios for GoodData.UI components. Tests are implemented using [Cypress](https://www.cypress.io/).
 There are two categories of tests: isolated and integrated. Isolated tests run against recorded backend responses, whereas integrated tests
-run against a live backend. Isolated tests run in the pre-merge phase and itegrated tests run in post-merge phase.
+run against a live backend. Isolated tests run in the pre-merge phase and integrated tests run in post-merge phase.
 
-## DEV Guide for isolated tests
+## Prepare env variables
```

Comment:
below this part, we should write a part to say create a reference work space and how to update the workspace fixture use **export-fixture** command. Can copy Reference workspace part from dashboard/analytical